database/engine/rrdengineapi.c

```diff
@@ -725,9 +738,11 @@ void rrdeng_get_35_statistics(struct rrdengine_instance *ctx, unsigned long long
     array[30] = (uint64_t)global_io_errors;
     array[31] = (uint64_t)global_fs_errors;
     array[32] = (uint64_t)rrdeng_reserved_file_descriptors;
-    array[33] = (uint64_t)ctx->stats.flushing_errors;
-    array[34] = (uint64_t)global_flushing_errors;
-    assert(RRDENG_NR_STATS == 35);
+    array[33] = (uint64_t)ctx->stats.pg_cache_over_half_dirty_events;
```

Comment:
There is a comment about that changing the indices can break user code. Does changing the use of index 33 from the number of flushing errors to the number of over_half_dirty events affect any user code?

Reply:
All corresponding statistics have been adjusted. The hazard of memory corruption is about the number of the actual statistics. Changing individual statistics only changes the semantics.

Reply:
Ah ha. I see that you have changed the number of the actual statistics as well...